code/index.html

```diff
@@ -1,30 +1,122 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>&#127795 Project 3</title>
+
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+  <!-- video or image as a header is cool :) -->
+
+  <header>
+    <div class="background">
+      <h2>The Gardeners</h2>
+    </div>
+
+  </header>
+
+  <section class="bottomPart">
+
+    <!-- sign-up form -->
+    <form action="http://httpbin.org/anything" method="POST">
+
+      <h3>FORM OF INTEREST</h3>
+
+      <div class="form-label">
+        <label>
+          Name
+          <input type="text" name="name" placeholder="name" required>
+        </label>
+
+        <label>
+          E-mail
+          <input type="email" name="email" placeholder="name@gmail.com">
+        </label>
+
+        <label>
+          Phone
+          <input type="tel" name="phonenumber" placeholder="706020833">
+        </label>
+      </div>
+
+
+      <!-- Radio -->
+
+      <label class="options">Gardening
+        <input type="checkbox" checked="checked">
+        <span class="checkmark"></span>
+      </label>
+
+
+      <label class="options">Pruning
+        <input type="checkbox">
+        <span class="checkmark"></span>
+      </label>
+
+      <label class="options">Landscaping
+        <input type="checkbox">
+        <span class="checkmark"></span>
+      </label>
+
+      <label class="options">Value pack (all of the above)
+        <input type="checkbox">
+        <span class="checkmark"></span>
+      </label>
+
+      <button id="button" class="btn" type="Submit">Send</button>
+
+      <h4>We will get back to you shortly!</h4>
+
+    </form>
+
+    <!-- Accordion -->
 
     <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
 
-      <div class="question" id="section1">Section 1</div>
+      <h1>INFO</h1>
```

Comment:
The `<h1>` should describe the whole page! I don't think "INFO" is suited as a `<h1>` but it would make a perfect `<h2>`!